tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -173,7 +173,7 @@
 
     <!-- Integration: Serilog -->
     <!-- Assembly: Serilog -->
-    <!-- Latest package https://www.nuget.org/packages/Serilog/2.12.0 -->
+    <!-- Latest package https://www.nuget.org/packages/Serilog/3.0.1 -->
     <PackageReference Include="Serilog" Version="2.12.0" />
```

Comment:
I think we need to bump this too?

Reply:
Hmm I must've missed this somehow when committing thanks for catching!

Reply:
I'm not entirely sure why, but for some reason this one wasn't being updated by the `GeneratePackageVersions` so I manually did it